app/boards/intel_adsp_ace30_ptl.conf

```diff
@@ -10,8 +10,11 @@ CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
```

Comment:
As Flavio wrote in this PR (https://github.com/thesofproject/sof/pull/9237), this symbol is deprecated.

Reply:
Done